test/TestConfig/settings.mk

```diff
@@ -232,7 +234,10 @@ setup_%:
 	@$(ECHO) set JCL_VERSION to $(JCL_VERSION)
 	@$(ECHO) set JAVA_BIN to $(JAVA_BIN)
 	@$(ECHO) set SPEC to $(SPEC)
-	@$(ECHO) Running $(TESTTARGET)
+	@$(ECHO) Running $(TESTTARGET) ...
+	@if [ $(TOTALCOUNT) -ne 0 ]; then \
+		$(ECHO) There are $(TOTALCOUNT) tests in $(TESTTARGET).; \
```

Comment:
I wonder if it would be better to say there are $(TOTALCOUNT) test **groups** in $(TESTTARGET), to avoid the questions around the fact that different systems use different nomenclature for tests, testcases, test suites, etc.  The nomenclature I have been using casually is that each test group counted in the TAP results is made up of zero or more test cases (though even that can be confusing as in playlist.xml file we say < test > < testCaseName >...

Real output:
16:26:38 [Test] Running sanity ...
16:26:38 [Test] There are 127 tests in sanity.   <-- this would be better to say There are 127 test groups in sanity.

As we can not count the number of test cases within a group prior to running them, without GREAT cost we should not pretend that we can.  I think the solution you have here is correct and honest.  

The PR plugin that is confusing the dev team by its output is falsely reporting number of test cases, as it is counting junit style tests, so it does not correctly report any commandline tester tests.  It should be made to be run in silent mode, since it is not properly counting our tests.

Reply:
Yes, we need to be more accurate on the wording here. The word **test** I was referring here is same as the one in TAP results. They are the test targets, for example, TestA_0 and TestB_1 are counted as 2 tests. So do you think **test target** is more accurate in this sense?